homeassistant/helpers/discovery.py

```diff
@@ -129,7 +129,7 @@ def load_platform(
     hass: core.HomeAssistant,
     component: Platform | str,
     platform: str,
-    discovered: DiscoveryInfoType,
+    discovered: DiscoveryInfoType | None,
```

Comment:
If you feel that `life360` should be adjusted instead to pass an empty dict, then I can adjust accordingly but I felt that since all the methods then use `discovery_info: DiscoveryInfoType | None = None` it was better to adjust here.

Reply:
> If you feel that life360 should be adjusted instead to pass an empty dict,

I don't feel anything about that. I was merely looking at this PR and I see a mismatch to this proposal and the function its calling. That's all.